zerver/openapi/zulip.yaml

```
@@ -2636,6 +2636,11 @@ paths:
                       type: string
                       description: |
                         The version of Zulip running in the server.
+                    zulip_feature_level:
+                      type: integer
+                      description: |
+                        An integer to indicate the features added to the server. It is
+                        finer-grained than the version number.
```

```suggestion
                        An integer to indicate the features added to the server. The
                        feature level increases monotonically: a value of N means the
                        server supports all API features introduced before feature level N.
                        This provides fine-grained distinctions among development versions
                        in between major releases.
```

(and similarly in the subsystem doc)

Basically I want to communicate the "monotonically increasing" part, because that's an important aspect of thinking about how to use this value.